graphics/managed_surface.cpp

```diff
@@ -386,18 +387,15 @@ void ManagedSurface::blitFromInner(const Surface &src, const Common::Rect &srcRe
 					bDest = bSrc;
 				} else {
 					// Partially transparent, so calculate new pixel colors
-					if (destFormat.bytesPerPixel == 2) {
-						uint32 destColor = *reinterpret_cast<uint16 *>(destVal);
-						destFormat.colorToARGB(destColor, aDest, rDest, gDest, bDest);
-					} else if (format.bytesPerPixel == 4) {
-						uint32 destColor = *reinterpret_cast<uint32 *>(destVal);
-						destFormat.colorToARGB(destColor, aDest, rDest, gDest, bDest);
-					} else {
-						aDest = 0xFF;
-						rDest = destVal[0];
-						gDest = destVal[1];
-						bDest = destVal[2];
```

Comment:
Why did you remove this case? This triggers when we have a 1bpp format

Reply:
`destFormat.bytesPerPixel` is not allowed to be 1 as per the assert in the beginning of the function.

Reply:
Ah sorry, I spoke too fast. It's allowed when formats are the same.

Reply:
But it's handled on line 339

Reply:
You are right, thanks @lephilousophe